tests/test_queues.py

```diff
@@ -223,10 +223,10 @@ def test_redis_handler():
     import redis
     from logbook.queues import RedisHandler
 
-    KEY = 'redis'
+    KEY = 'redis-' + str(os.getpid())
```

Comment:
I ran the tests with `tox -p auto`, which makes runs multiple test envs in parallel.
These changes make the tests not write to the same keys.

Reply:
Please change to `KEY = 'redis-{}'.format(os.getpid())`